test-e2e-re/lib/FixtureUtils.re

```diff
@@ -0,0 +1,84 @@
+module Path = EsyLib.Path;
+module Option = EsyLib.Option;
+module Result = EsyLib.Result;
+module System = EsyLib.System;
+module Dir = Bos.OS.Dir;
+module File = Bos.OS.File;
+module BPath = Bos.OS.Path;
+
+open Result.Syntax;
+
+type fixtureCopy = {
+  name: string,
+  path: string,
+};
+
+type fixtureFile = {
+  name: string,
+  data: string,
+};
+
+type fixtureDir('t) = {
+  name: string,
+  items: list('t),
+};
+
+type fixture =
+  | FixtureSymlink(fixtureCopy)
+  | FixtureFileCopy(fixtureCopy)
+  | FixtureFile(fixtureFile)
+  | FixtureDir(fixtureDir(fixture));
+
+let transfer = (ic, oc, ()) => {
+  let chunkSize = 1024 * 1024; /* 1mb */
+  let buffer = Bytes.create(chunkSize);
+  let rec loop = () => {
+    switch (input(ic, buffer, 0, chunkSize)) {
+    | 0 => Result.return()
```

Comment:
`Result.Syntax` module which opened above contains `return` so it can be just `return()`:
```suggestion
    | 0 => return()
```